os/kernel/pthread/pthread_mutexunlock.c

```diff
@@ -83,6 +84,34 @@
 /****************************************************************************
  * Private Functions
  ****************************************************************************/
+/****************************************************************************
+ * Name: pthread_mutex_unlock
+ *
+ * Description:
+ *   Return true is the mutex is locked.
+ *
+ * Parameters:
+ *   None
+ *
+ * Return Value:
+ *   Returns true if the mutex is locked
+ *
+ ****************************************************************************/
+
+static inline bool pthread_mutex_islocked(FAR struct pthread_mutex_s *mutex)
```

Comment:
In pthread_mutex_unlock, pthread_mutex_t is used for mutex variable. Even though those are same but how about keep same type?